src/kbmod_wf/ic_workflow.py

```diff
@@ -0,0 +1,260 @@
+import argparse
+import os
+import toml
+import parsl
+from parsl import python_app, File
+import parsl.executors
+
+from kbmod_wf.utilities.configuration_utilities import apply_runtime_updates, get_resource_config
+from kbmod_wf.utilities.executor_utilities import get_executors
+from kbmod_wf.utilities.logger_utilities import configure_logger
+
+
+@python_app(
+    cache=True,
+    executors=get_executors(["local_dev_testing", "local_thread"]),
+    ignore_for_cache=["logging_file"],
+)
+def create_uri_manifest(inputs=[], outputs=[], runtime_config={}, logging_file=None):
```

Comment:
I believe that I have addressed the `list` issue everywhere. I have yet to go around and add docstrings. 

As a next step, I would like to do a bit of code organization now that there are multiple workflows defined so that we can reuse the app definitions between the workflows and reduce the code duplication.